app/webpacker/css/admin_v3/globals/palette.scss

```diff
@@ -5,3 +5,5 @@ $color-3: #008397 !default; // Teal (Allports)
 $color-4: #6788a2 !default; // Dark Blue
 $color-5: #c85136 !default; // Red/Orange (Mojo)
 $color-6: #ff9300 !default; // Yellow
+$color-7: #EFF1F2 !default; // Light grey
```

Comment:
This file is no more prettified now ;( (I can see because of the uppercase on html color code)